content/contribute.fr.md

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
Note to myself: We can now link this page in the footer.

Reply:
I updated the `menus.fr.toml` idk if it's enough.